modules/external_state/external-state.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "external-state"
+linkTitle: "external-state"
+date: 2021-07-21
+draft: false
+weight: 1
+description: Adds the ability to refer to external, preexisting terraform states in opta.
```

Comment:
random question, can this module be used when referring to a stack managed by opta?

I don't know if it's an interesting use case.

Reply:
`(e.g. pre-existing
or created outside of opta, or another Opta service/environment)`